js/default-configs/layouts-config.js

```diff
@@ -221,7 +221,7 @@ window.flListLayoutConfig = {
   'small-h-card': {
     'filtersEnabled': false,
     'filterFields': [],
-    'chatEnabled': true,
+    'chatEnabled': false,
```

Comment:
Why is this `false`?

Reply:
@tonytlwu I thought that we want to add this option as we did with sorting, allowing users to active it by themself.
Should we enable the chat option from the start?

Reply:
I see. Then yes, `false` is fine 🆗